src/composers/field.tsx

```diff
@@ -59,10 +62,11 @@ export function Field<Props>(WrappedComponent: IncomingField<Props>,
         }
 
         public render() {
+            const fieldState = this.props.form.getFieldState(this.getFieldName());
```

Comment:
Good question. This is following the initial design we had and maybe should change. The `getFieldState` gets called in favor of the `getValidationError` we had before. Do we wan't to store the validationError localy also?